spec/spec_helper_common.rb

```diff
@@ -134,4 +114,5 @@ def reset(object)
 end
 
 # I don't think this is necessary anymore with the latest factory_girl
```

Comment:
Can this comment be removed now?

Reply:
Yes, it can. Good catch!

I have removed it.